CorsixTH/CMakeLists.txt

```diff
@@ -14,18 +14,21 @@ if(MSVC)
   add_definitions(-D_CRT_SECURE_NO_WARNINGS)
 endif()
 
+set(CORSIX_TH_SEARCH_LOCAL_DATADIRS ${SEARCH_LOCAL_DATADIRS)
```

Comment:
missing closing }

Reply:
typo from your comment :( nevertheless I'll fix it later (busy now